server/game/cards/01_SOR/MedalCeremony.ts

```diff
@@ -0,0 +1,42 @@
+import AbilityHelper from '../../AbilityHelper';
+import { UnitCard } from '../../core/card/CardTypes';
+import { EventCard } from '../../core/card/EventCard';
+import { TargetMode, Trait } from '../../core/Constants';
+import { AttacksThisPhaseWatcher } from '../../stateWatchers/AttacksThisPhaseWatcher';
+import { StateWatcherRegistrar } from '../../core/stateWatcher/StateWatcherRegistrar';
+
+export default class MedalCeremony extends EventCard {
+    private attacksThisPhaseWatcher: AttacksThisPhaseWatcher;
+
+    protected override getImplementationId() {
+        return {
+            id: '4536594859',
+            internalName: 'medal-ceremony',
+        };
+    }
+
+    protected override setupStateWatchers(registrar: StateWatcherRegistrar) {
+        this.attacksThisPhaseWatcher = AbilityHelper.stateWatchers.attacksThisPhase(registrar, this);
```

Comment:
I think that's the purpose of the `AbilityHelper` export, since you can just go into the StateWatcherLibrary.ts file and see them listed there (same as for GameSystemLibrary, OngoingEffectLibrary, CostLibrary). Any reason to do this one differently?

Reply:
@AMMayberry1 I was more thinking that having to declare the watcher on the card was a bit of an annoyance. It's not a must-have. If we decide to refactor it down the line it wouldn't be too bad.

Reply:
Oh yeah I could see that. Generally I’m trying to cut back on long lists of properties that are commonly null. Since it would be an easy change like you say leave it this way for now and see how it looks over time.